.travis.yml

```diff
@@ -1,5 +1,14 @@
-sudo: false
-language: node_js
+sudo: required
+dist: trusty
+language: php
```

Comment:
It tests if the app can be enabled on master and stable9 and if it works on 5.6 and 7. Future tests will be easier to add, I don't see a downside :)